main.js

```diff
@@ -0,0 +1,30 @@
+var carIcon = document.querySelector('img');
+
+// var carDirection = [
+//   {
+//     direction: 'left'
+//   },
+//   {
+//     direction: 'right'
+//   },
+//   {
+//     direction: 'up'
+//   },
+//   {
+//     direction: 'down'
+//   }
+// ];
+
+document.addEventListener('keydown', turnCar);
+
+function turnCar(event) {
+  if (event.code === 'ArrowLeft') {
+    carIcon.className = 'left car';
+  } else if (event.code === 'ArrowRight') {
+    carIcon.className = 'left right car';
```

Comment:
Shouldn't this just be `right car`?

Reply:
That was a mistake from when I was troubleshooting something, it is fixed now, thanks.